DXMainClient/DXGUI/Multiplayer/CnCNet/CnCNetGameLoadingLobby.cs

```diff
@@ -652,9 +652,9 @@ protected override void UpdateDiscordPresence(bool resetTimer = false)
                 return;
 
             PlayerInfo player = Players.Find(p => p.Name == ProgramConstants.PLAYERNAME);
-            if (player == null || lblMapNameValue == null || lblGameModeValue == null)
+            if (player == null || String.IsNullOrEmpty(lblMapNameValue.Text) || String.IsNullOrEmpty(lblGameModeValue.Text))
```

Comment:
The values of lblMapNameValue and lblGameModeValue should never be null or empty (since SetUp sets them when the lobby is opened), so I don't think it's useful to check for them.